src/allmydata/web/unlinked.py

```diff
@@ -160,7 +163,6 @@ def _then_redir(res):
             new_url = "uri/" + urllib.quote(res.get_uri())
             req.setResponseCode(http.SEE_OTHER) # 303
             req.setHeader('location', new_url)
-            req.finish()
             return ''
```

Comment:
Pretty sure it was always wrong to call `req.finish()` and then also return a string.